scripts/benchmark/benchmark-build-speed

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
I suggest adding an extension ".sh" to be clear it's a shell script without the need to read it